src/components/Header.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { RefObject } from 'react';
+
+type Props = {
+  newTodoField: RefObject<HTMLInputElement>;
+  title: string;
+  setTitle: (title: string) => void;
+  handleAddToto: (title: string) => void;
+};
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
+export const Header: React.FC<Props> = ({
```

Comment:
if you pass ref - you have to wrap your component with `forwardRef()` and take it as second argument 
```suggestion
export const Header: React.FC<Props> = forwardRef((props, passedRef) => ...jsx)
```